lib/runner/extract-runnable-items.js

```diff
@@ -186,6 +188,15 @@ var sdk = require('postman-collection'),
 
         items = findItemsOrGroups(collection, entrypointLookup, true);
 
+        // Extracting the items and folders by the order in which they appear as folder/request arguments,
+        // only if specified in the entrypoint.preserveOrder
+        if (preserveOrder && items && items.reference && Array.isArray(entrypoints)) {
+            entrypoints.forEach((item) => {
+                itemsInOrder && itemsInOrder.push(items.reference[item]);
+            });
+            items.members = itemsInOrder;
```

Comment:
```suggestion
            items.members = entrypoints.map(function (ref) {
                return items.reference[ref];
            });
```

This can be simplified by using the `map` instead of swapping arrays.